backend/capellacollab/sessions/hooks/session_token.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+
+import datetime
+
+from capellacollab.permissions import models as permissions_models
+from capellacollab.users.tokens import crud as tokens_crud
+
+from . import interface
+
+
+class SessionTokenIntegration(interface.HookRegistration):
+    """Create a PAT valid for the duration of the session."""
+
+    def configuration_hook(
+        self,
+        request: interface.ConfigurationHookRequest,
+    ) -> interface.ConfigurationHookResult:
+        token, password = tokens_crud.create_token(
```

Comment:
Is password the right name here?

Reply:
It's the actual `token` that can be used against the API. Since basic authentication consists of username and password, I'd consider password as correct. It also avoids confusion with the actual token database object.